maestro-client/src/test/java/maestro/android/AndroidLaunchArgumentsTest.kt

```diff
@@ -14,7 +14,8 @@ class AndroidLaunchArgumentsTest {
             "isMaestro" to true,
             "cartValue" to 4,
             "cartValueDouble" to 4.4,
-            "cartColor" to "Hello this is cart value which is orange"
+            "cartColor" to "Hello this is cart value which is orange",
+            "cartTimeStamp" to 1683113805263
```

Comment:
Could we add another test for `0` to ensure that gets parsed correctly?

Reply:
Done ✅